resource/js/tab-hierarchy.js

```diff
@@ -148,6 +153,13 @@ const tabHierApp = Vue.createApp({
         height: 'calc( 100% - ' + height + 'px)',
         width: width + 'px'
       }
+    },
+    compareLabels (a, b) {
+      const labelA = a.label ? a.label : a.prefLabel
+      const labelB = b.label ? b.label : b.prefLabel
+      const lang = window.SKOSMOS.content_lang ? window.SKOSMOS.content_lang : window.SKOSMOS.lang
+
+      return labelA.localeCompare(labelB, lang)
```

Comment:
This may fail if labelA happens to be null, which currently happens for example in some of the Cypress tests. I suggest checking for the case where either labelA or labelB is null and using another value instead such as "" (empty string).